src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Product.xml

```diff
@@ -129,6 +129,14 @@
         </itemOperations>
 
         <subresourceOperations>
+            <subresourceOperation name="images_get_subresource">
+                <attribute name="method">GET</attribute>
+                <attribute name="path">/admin/products/{code}/images</attribute>
+                <attribute name="openapi_context">
+                    <attribute name="summary">Retrieves the collection of ProductImage resources for a given Product.</attribute>
```

Comment:
Summary should be written in natural language.

![image](https://github.com/Sylius/Sylius/assets/870747/e2b445c2-3f25-4139-8c55-0ac428ef23ef)
![image](https://github.com/Sylius/Sylius/assets/870747/87b1b511-0ad0-4b43-b344-4c1fd4d4e8d4)


```suggestion
                    <attribute name="summary">Retrieves a collection of product images for a given product</attribute>
```

But I'm not sure if custom summary is even needed here, auto-generated summaries looks good enough on regular CRUD actions.

Reply:
There is a need for custom summary, because, as it is a subresource, the auto-generated summary looks like:
<img width="1359" alt="image" src="https://github.com/Sylius/Sylius/assets/6140884/3fb7bca6-1e9a-498f-a6fb-e201530a0186">
My proposal is as close as possible to the auto-generated version, so I would leave it is now.